packages/expo-ui/ios/SwitchView.swift

```diff
@@ -22,9 +23,11 @@ struct SwitchView: ExpoSwiftUI.View {
         if props.checked == newValue {
           return
         }
-        props.onCheckedChanged([
+        let payload = [
           "checked": newValue
-        ])
+        ]
+        props.onCheckedChanged(payload)
```

Comment:
why do you need to do it like so? could you maybe create a mock envobject same as we do for nativenodeproxy now?